io/io/src/TBufferFile.cxx

```diff
@@ -1328,16 +1492,31 @@ void TBufferFile::ReadFastArray(Short_t *h, Int_t n)
    if (n <= 0 || l > fBufSize) return;
 
 #ifdef R__BYTESWAP
+   if (IsBufBigEndian()) {
 # ifdef USE_BSWAPCPY
-   bswapcpy16(h, fBufCur, n);
-   fBufCur += sizeof(Short_t)*n;
+      bswapcpy16(h, fBufCur, n);
```

Comment:
github shows this white space as darker green.  I take it that this __might__ be meaning there is a tab characters.  tab characters are *not* allowed in source code has their actual size depend on the user's editor setting and mess up the alignment.